### PR TITLE
Minor edits to Git lesson

### PR DESCRIPTION
**Pull request by aays**
Just a bit of cleanup - typos, edits for flow, and better paragraph formatting. I also noticed many instances last year of students being confused by the `[filename]` syntax, thinking that the `[]` were actually part of the command, so I've tried clarifying that through the lesson.

### CODE REVIEW

File: lec15-git-projects.Rmd [hunk 1]
```diff
@@ -29,121 +29,133 @@ author: Ahmed Hasan
...
-below_)
-
-To install Git (and Bash) on **Windows**, download [Git for Windows](https://gitforwindows.org/) from its [website](https://github.com/git-for-windows/git/releases/tag/v2.19.1.windows.1). 
+Git is primarily used on the command line. The implementation of the command
+line that we'll be using is known as a 'bash interpreter', or 'shell'. While
+bash interpreters are natively available on Mac and Linux operating systems,
+Windows users may have to externally install a bash shell.
+
+To install Git on a **Mac**, you will have to install
+[XCode](https://itunes.apple.com/ca/app/xcode/id497799835?mt=12), which can be
```
Comment (QuLogic) [inline lec15-git-projects.Rmd]: Wiki seems to have renamed it XCode->Xcode.

File: lec15-git-projects.Rmd [hunk 2]
```diff
@@ -29,121 +29,133 @@ author: Ahmed Hasan
...
-This is done with the `git config` command. When using Git, _all_ commands begin
-with `git`, followed by more specific subcommands. 
+This is done with the `git config` command. When using Git, _all_ commands
+begin with `git`, followed by more specific subcommands. 
 
 ```
-git config --global user.name "Your Name"
-git config --global user.email "youremail@email.com"
+git config --global user.name "My Name"
+git config --global user.email "myemail@email.com"
```
Comment (QuLogic) [inline lec15-git-projects.Rmd]: We should make this `@example.com` now.

File: lec15-git-projects.Rmd [hunk 3]
```diff
@@ -153,17 +165,21 @@ Here, we'll use `nano`, as mentioned above:
 git config --global core.editor "nano"
 ```
 
-here is a [link](http://swcarpentry.github.io/git-novice/02-setup/index.html) to instructions on how to config other text editors. 
+If you would like to set up a different text editor as your default, see
+[here](http://swcarpentry.github.io/git-novice/02-setup/index.html).
```
Comment (QuLogic) [inline lec15-git-projects.Rmd]: https now.

File: lec15-git-projects.Rmd (hunk 1 above)
Comment (aays) [reply lec15-git-projects.Rmd]: Dammit, Apple...

**Pull request merged**